### ISSUE TITLE
Launching LabCompass while inside Lab, no way to assign position

### ISSUE THREAD

**Issue by Freezy3** [yznpku/LabCompass#39]
Started a Lab and forgot to launch LabCompass, with it open it won't auto detect and can't find any way or any comments on how to assign where your positioned.

**Comment by miaXcova** [comment]
Agreed, this would be an awesome QoL addition.

**Comment by valerian** [comment]
Yes please, more than half of the time I start the lab and only after I think about launching LabCompass and/or downloading the latest map
